configs/rules.json

```diff
@@ -419,6 +419,10 @@
     "confidence_threshold": 0.95,
     "cc": []
   },
+  "performancebug": {
+    "days_lookup": 7,
+    "confidence_threshold": 0.9
```

Comment:
It would be nice to move `confidence_threshold` to be a constructor parameter. See: https://github.com/mozilla/bugbot/issues/1437